src/utils/SpokeUtils.ts

```diff
@@ -316,11 +290,12 @@ export async function fillStatusArray(
  * @returns The block number at which the relay was completed, or undefined.
  */
 export async function findFillBlock(
-  spokePool: Contract,
+  spokePoolClient: SpokePoolClient,
```

Comment:
This function currently isn't used anymore - I think we have future plans for it though so we can keep it around. cc @pxrl

Reply:
Can we also revert this change? I'd really like to avoid forcing instantiation of a SpokePoolClient instance just to query the the fill status at a given spoke.